frontend/src/main/components/Review/ReviewTable.js

```diff
@@ -47,8 +47,7 @@ export default function ReviewTable({ reviews, currentUser }) {
         {
             Header: 'comments',
             accessor: 'comments',
-        }
-        
+        } 
```

Comment:
on line 12 of this file - the url should say `review` not `reviews`. currently this does not match the existing routing.